cvat-core/src/session.js

```diff
@@ -2017,6 +2028,7 @@
                 bugTracker: false,
                 subset: false,
                 labels: false,
+                project_id: false,
```

Comment:
Should not ``updatedFields`` have a method ``reset()`` that goes over the boolean properties and set them to false?